### ISSUE TITLE
Support for CZI attachments

### ISSUE THREAD

**Issue by ChiliJohnson** [AllenCellModeling/aicspylibczi#40]
Is there any planned support for retrieving CZI attachments / is it within the scope of this library to provide access to CZI attachments?

Just being able to retrieve raw attachment bytes would be very helpful. After a cursory glance it appears `libczi` [supports reading raw attachment bytes](https://github.com/zeiss-microscopy/libCZI/blob/master/Src/libCZI/CziAttachment.h)

**Comment by JacksonMaxfield** [comment]
@heeler Would have more insight. I would assume it's probably in the scope of this package personally but wait for his response.

**Comment by heeler** [comment]
I'm aware of attachments in the subblocks but I'm not entirely sure what is in them. I haven't seen any examples in our institute. Can you explain the use case and do you have an example file you'd be willing to share? Thanks!

**Comment by ChiliJohnson** [comment]
Of course! I have some CZI slide scans produced with a Zeiss Axio Scan.Z1 which, in addition to the pixels stored in the body of the CZI, also attaches two additional images: one "preview" image of the entire slide, and one "label" image of the slide label area (where we attach barcodes). Both of these (but the preview image most of all) are useful in determining the identity of the samples which were imaged.

It appears that these attachments live in the `ZISRAWATTACH` segments, with the names `SlidePreview`, and `Label`, and are CZI images themselves. Here's [an example image](https://s3-us-west-2.amazonaws.com/external-share.system1bio/aicspylibczi/CAC5CBF9-0E98-4C2F-8FC8-D95BEA4D2637.czi) which contains both of these attachments.

I should note that this image contains other attachments in various formats that might be useful to others, but the two that I mentioned are just the two I care about for my use case.

Let me know if you'd like any more context, I'd be happy to discuss!
